src/components/WeeklyCalendar/helpers/getFirstDayOfWeek.js

```diff
@@ -0,0 +1,13 @@
+export default function getFirstDayOfWeek(date) {
+    const clone = new Date(date);
+    const isValidDate = !isNaN(clone.getTime());
+    if (isValidDate) {
+        clone.setDate(clone.getDate() - clone.getDay());
+        clone.setHours(0, 0, 0, 0);
+        return clone;
+    }
+    const today = new Date();
+    today.setDate(today.getDate() - today.getDay());
+    today.setHours(0, 0, 0, 0);
+    return today;
```

Comment:
since minWeek, maxWeek can be Infinity, -Infinity you may want to handle those case here and return back the same value was passed